LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -246,10 +243,6 @@ public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaReques
 
                         loRaDevice.SetFcntUp(payloadFcntAdjusted);
                     }
```

Comment:
You removed here the line
```c#
Logger.Log(loRaDevice.DevEUI, $"invalid frame counter, msg: {payloadFcntAdjusted} server: {loRaDevice.FCntUp}", LogLevel.Information);
```

Does it mean we are not logging if the fctnUp is incorrect? Example: unconfirmed up where the fcntUp < server fcntUp

Reply:
By running the test When_Upstream_Fcnt_Is_Lower_Or_Equal_To_Device_Should_Discard_Message,  you get the following logs (first one device fcnt up = 10 / server :10, second device fcnt up = 9/ server :10 ): 
2019-03-22 10:42:04.530 0000000000000001: device in cache
2019-03-22 10:42:04.544 0000000000000001: Converted 16bit FCnt 10 to 32bit FCnt 10
2019-03-22 10:42:04.548 0000000000000001: invalid frame counter, message ignored, msg: 10 server: 10
2019-03-22 10:42:04.553 0000000000000001: processing time: 00:00:00.0394948

and 

2019-03-22 10:42:36.304 0000000000000001: device in cache
2019-03-22 10:42:36.318 0000000000000001: Converted 16bit FCnt 9 to 32bit FCnt 9
2019-03-22 10:42:36.323 0000000000000001: invalid frame counter, message ignored, msg: 9 server: 10
2019-03-22 10:42:36.337 0000000000000001: processing time: 00:00:00.0511546

I could reput the line but I am not sure it is necessary

Reply:
this log is done in the validateRequest method

Reply:
Ah, that is good then. Thanks